### PR TITLE
add support for loongarch64

### PR DESCRIPTION
**Pull request by wuruilong01**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ x] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by tautschnig** [comment]
Patch is now included in cbmc 5.95.1-5 as just uploaded to Debian's build queue. I'll check https://buildd.debian.org/status/package.php?p=cbmc&suite=sid the next hours to confirm that this builds fine for loong64.

**Comment by tautschnig** [comment]
Builds and tests appear to have succeeded: https://buildd.debian.org/status/fetch.php?pkg=cbmc&arch=loong64&ver=5.95.1-5&stamp=1714405344&raw=0

File: src/util/config.cpp [hunk 1]
```diff
@@ -756,6 +785,8 @@ void configt::set_arch(const irep_idt &arch)
     ansi_c.set_arch_spec_x86_64();
   else if(arch=="i386")
     ansi_c.set_arch_spec_i386();
+  else if(arch=="loongarch64")
```
Comment (tautschnig) [inline src/util/config.cpp]: Please add spaces around `==`

Comment (wuruilong01) [reply src/util/config.cpp]: Code has been modified to handle

**Review by tautschnig — APPROVED** [review]

**Pull request merged**